### PR TITLE
Tonyalpha/package upgrades

### PR DESCRIPTION
**Pull request by okhaimie-dev**
## How Has This Been Tested?

Ran code to make sure there are no new breaking changes. 
Tested the app functionalities on browser.

### CODE REVIEW

**Comment by okhaimie-dev** [comment]
This PR contains major dependencies upgrades. 

ReactJS
Types/Node
NFT.Storage
ReactDOM
Jest
TS-Jest

And some minor upgrades. Currently there are no breaking changes to the best of my knowledge, we might want to do a little more check while I do extra readings. :)

**Comment by aahna-ashina** [comment]
The menu layout looks broken on mobile:  https://app-git-fork-okhaimie-dev-tonyalpha-package-upgrades-nation3.vercel.app/

image

File: ui/package.json [hunk 1]
```diff
@@ -16,36 +16,36 @@
   "dependencies": {
     "@heroicons/react": "^2.0.11",
     "@metamask/eth-sig-util": "^5.0.0",
-    "@types/node": "^17.0.34",
-    "@types/react-dom": "^18.0.4",
-    "daisyui": "^2.13.5",
+    "@types/node": "^18.7.23",
+    "@types/react-dom": "^18.0.6",
+    "daisyui": "^2.31.0",
```
Comment (aahna-ashina) [inline ui/package.json:21]: I think this upgrade caused the menu layout to change?

**Review by aahna-ashina — CHANGES_REQUESTED** [review]

File: ui/package.json (hunk 1 above)
Comment (okhaimie-dev) [reply ui/package.json:21]: I'll look into this. And make changes accordingly.

**Comment by aahna-ashina** [comment]
@okhaimie-dev Now that we are downgrading wagmi to solve the passport claim problem, I suggest we merge the branch at https://github.com/nation3/app/pull/140 _before_ this one.

**Comment by aahna-ashina** [comment]
@okhaimie-dev #140 has been merged now 👍

**Comment by aahna-ashina** [comment]
@okhaimie-dev Closed this, as it's out-of-date.

**Pull request closed without merging**